src/components/pages/login/LoginForm.js

```diff
@@ -1,18 +1,33 @@
 import React, { useState } from 'react'
+import { useNavigate } from 'react-router-dom';
 
 export default function LoginForm() {
 
     // state
     const [newName, setNewName] = useState("");
+        // is authorized to access the order page
+        const [goToOrderPage, setGoToOrderPage] = useState(false);
+        // users account
+        const users = [
+            {username: 'bob'},
+            {username: 'claire'}
+        ];
+    const account = users.find((user) => user.username === newName);
+    const navigate = useNavigate();
 
     // comportements
 
     // form submission
     const handleSubmit = (event) => {
         event.preventDefault();
-        alert(`Bonjour ${newName}`);
-        // input clear
-        setNewName("");
+        // redirection order page
+        if(goToOrderPage) {
+            navigate(`/order/${newName}`);
+        } else {
```

Comment:
```suggestion
        } else {
```

QUASIMENT tous les else sont inutiles.